core/data/tests/can_generate_const/output.go

```diff
@@ -0,0 +1,5 @@
+package proto
+
+import "encoding/json"
+
+const MyVar uint32 = 12;
```

Comment:
I would not expect a semicolumn here:
```suggestion
const MyVar uint32 = 12
```